macros/models/_project/paraswap/v6/paraswap_v6_balancer_v2_method.sql

```diff
@@ -0,0 +1,101 @@
+{% macro paraswap_v6_balancer_v2_method(tableOuter, tableInner, srcTable, inOrOut, method ) %}
+WITH
+                  {{ tableOuter }} as (
+                    WITH
+                      {{tableInner}} AS (
```

Comment:
context for line 5 to 34:

Our contract has a function `swapExactAmountInOnBalancerV2(tuple balancerData,uint256 partnerAndFee,bytes permit,bytes data)` where data is expected to be the abi encoding of a call to balancer vault function (say `batchSwap(data..)`), in order to have access to srcToken, destToken, fromAmount and destAmount from our contract to perform actions (transfer, checks etc...) we abi decode the balancer call params. As we don't have access to this capability in dune, we had to do this manually.

sample: https://polygonscan.com/tx/0x689415447f5770031d271a61a7a98c4ca246c644a4e16fb5a793dd0c9ae5f8a0


same applies to swapExactAmountOutOnBalancerV2